pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -69,13 +72,19 @@ func (p *podAffinityPriorityMap) processTerm(term *v1.PodAffinityTerm, podDefini
 	if err != nil {
 		return err
 	}
+	if fixedNode.Labels == nil {
```

Comment:
Prefer to `len(fixedNode.Labels) == 0` to tolerate the case it's inited with an empty map.

Reply:
done